server/routes/user.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import Helper from '../helpers';
+import OrderController from '../controllers/orderController';
+
```

Comment:
I'd suggest you destructure out methods from the classes rather than using dot notation.